projects/learned_inhibitory_connections_sp/setup.py

```diff
@@ -0,0 +1,11 @@
+
+from setuptools import setup, find_packages
+
+setup(name='LateralPooler',
```

Comment:
I think you might need this argument as well:

```
package_namespace=['lateralpooler'],
```

Reply:
We shouldn't have setup.py at all - why do we need it?  The imported code should go into the htmresearch directory structure.